README.md

```diff
@@ -50,21 +50,26 @@ import PropTypes from 'prop-types';
 class AssetPathProvider extends Component {
   getChildContext() {
     return {
-      assetBasePath: '',
+      assetBasePath: 'assets/',
     };
   }
 
   render() {
     const { children } = this.props;
-    return Children.only(children);
+
+    return (
+      <div>
```

Comment:
Why change this?

Reply:
In the current state, a `div` wrapper is implied which is also not ideal but may be less confusing